README.md

```diff
@@ -32,8 +32,8 @@ This C# SDK is automatically generated by the [Swagger Codegen](https://github.c
 - SDK version: 2.3.1
 - Build package: io.swagger.codegen.v3.generators.dotnet.CSharpClientCodegen
 
-<a name="frameworks-supported"></a>
-## Frameworks supported
+<a name="requirements"></a>
+## Requirements
 - .NET 5.0 or later
 
 <a name="dependencies"></a>
```

Comment:
Please add a Feedback and Support section consistent with https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk#support-and-feedback